### PR TITLE
Added DApps Store swarm

### PR DESCRIPTION
**Pull request by jeluard**
WIP, waiting for feedback from @andytudhope

### CODE REVIEW

File: ideas/315-dapps-store.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+- Andy
+- Julien
+- ?
+
+(+ Janitor swarm)
+
+## Communication
+
+`status channel`: #315-dapps-store
+`sync schedule`: weekly, during core chat sync
```
Comment (oskarth) [inline ideas/315-dapps-store.md]: Why chat?

File: ideas/315-dapps-store.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+`Meeting notes`: 
+
+## Research
+
+`Timebox`: wrap by 21/12
```
Comment (oskarth) [inline ideas/315-dapps-store.md]: Seems too tight, Andy isn't back until 19th ish.

File: ideas/315-dapps-store.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+- I want to vote for a DApp inclusion.
+--
+- As a DApp developer:
+- I want to be able to propose my DApp for inclusion.
+- I want to vote for my DApp inclusion.
+
+
+`Research questions`:
+
+- Is it possible to
```
Comment (oskarth) [inline ideas/315-dapps-store.md]: Yes

File: ideas/315-dapps-store.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+`Meeting notes`: 
+
+## Research
+
+`Timebox`: wrap by 21/12
+
+`Previous work`:
+
+DApps are currently exposed as a static catalog hardcoded in Status client.
```
Comment (oskarth) [inline ideas/315-dapps-store.md]: State of the dapps as well

And link to Andy's proposal with SNT utility

Comment (oskarth) [reply ideas/315-dapps-store.md]: Here https://discuss.status.im/t/how-to-curate-dapps-simply/759

File: ideas/315-dapps-store.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+The DApps Store swarm will deliver SNT based curation mechanisms in accordance with the white paper.
+
+It allows individuals to curate DApps by staking SNT and have access to this list within Status.
+
+## Contributors
+
+- Andy
+- Julien
+- ?
```
Comment (oskarth) [inline ideas/315-dapps-store.md]: Need design

**Comment by oskarth** [comment]
As part of the research phase, we need to pair that UI down to solve minimal set of user stories. Reasonable outcome would be set of basic wireframes.

File: ideas/315-dapps-store.md (hunk 5 above)
Comment (oskarth) [reply ideas/315-dapps-store.md]: This is Andrei

File: ideas/315-dapps-store.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
+---
+id: #315-dapps-store
```
Comment (oskarth) [inline ideas/315-dapps-store.md]: will have to be changed, along with filename

File: ideas/315-dapps-store.md (hunk 1 above)
Comment (jeluard) [reply ideas/315-dapps-store.md]: Oops

File: ideas/315-dapps-store.md (hunk 3 above)
Comment (jeluard) [reply ideas/315-dapps-store.md]: Legit

**Pull request merged**